docs/guide/extending_modules.md

```diff
@@ -131,20 +131,113 @@ The following props are deprecated:
 
 ### Configuration
 
-#### Search limits
-
-You can configure search limits in Universal Discovery module in `universal_discovery_widget.yml` file, under the following path:
+You can configure Universal Discovery module in [universal_discovery_widget.yml file.](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/src/bundle/Resources/config/universal_discovery_widget.yml)
+There you can set e.g. visible tabs, allowed Content Types, search limits etc.
 
 ```yaml
 system:
     default:
```

Comment:
```suggestion
    <siteaccess>:
```
As used in other SA-aware config examples

Reply:
Actually this example shows config from this file: https://github.com/ezsystems/ezplatform-admin-ui/blob/master/src/bundle/Resources/config/universal_discovery_widget.yml
This is not part of below tutorial just basic config file.